tools/website-test/Cargo.toml

```diff
@@ -18,7 +18,7 @@ wasm-bindgen-futures = "0.4"
 weblog = "0.3.0"
 yew = { path = "../../packages/yew/", features = ["ssr"] }
 yew-router = { path = "../../packages/yew-router/" }
-tokio = { version = "1.15.0", features = ["full"] }
+#tokio = { version = "1.15.0", features = ["full"] }
```

Comment:
Why is this commented out? Shouldn't it be used in this example? If its not used maybe just remove the line?

Reply:
Testing purposes. I was figuring out why CI is failing.